src/_nebari/subcommands/init.py

```diff
@@ -599,7 +533,7 @@ def init(
         )
 
         # Digital Ocean deprecation warning -- Nebari 2024.7.1
-        if inputs.cloud_provider == ProviderEnum.do.value.lower():
+        if inputs.cloud_provider == "do":
```

Comment:
Should this not say that it has been deprecated and nebari no longer supports this option?

Reply:
Good catch, thanks. For some reason I though this was part of an upgrade step but that's not the case.